huaweicloud/services/lts/resource_huaweicloud_lts_cce_access.go

```diff
@@ -0,0 +1,534 @@
+			"exclude_labels":       utils.PathSearch("access_config_detail.excludeLabels", resp, nil),
+			"log_envs":             utils.PathSearch("access_config_detail.logEnvs", resp, nil),
+			"include_envs":         utils.PathSearch("access_config_detail.includeEnvs", resp, nil),
+			"exclude_envs":         utils.PathSearch("access_config_detail.excludeEnvs", resp, nil),
+			"log_k8s":              utils.PathSearch("access_config_detail.logK8s", resp, nil),
+			"include_k8s_labels":   utils.PathSearch("access_config_detail.includeK8sLabels", resp, nil),
+			"exclude_k8s_labels":   utils.PathSearch("access_config_detail.excludeK8sLabels", resp, nil),
+		},
+	}
+}
+func resourceCceAccessConfigUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+
+	updateCceAccessConfigChanges := []string{
+		"access_config",
+		"host_group_ids",
+		"tags",
+		"binary_collect",
+		"log_split",
+	}
+
+	if d.HasChanges(updateCceAccessConfigChanges...) {
+		var (
+			updateCceAccessConfigHttpUrl = "v3/{project_id}/lts/access-config"
+			updateCceAccessConfigProduct = "lts"
+		)
+		ltsClient, err := cfg.NewServiceClient(updateCceAccessConfigProduct, region)
+		if err != nil {
+			return diag.Errorf("error creating LTS client: %s", err)
+		}
+
+		updateCceAccessConfigPath := ltsClient.Endpoint + updateCceAccessConfigHttpUrl
+		updateCceAccessConfigPath = strings.ReplaceAll(updateCceAccessConfigPath, "{project_id}", ltsClient.ProjectID)
+
+		updateCceAccessConfigOpt := golangsdk.RequestOpts{
+			KeepResponseBody: true,
+		}
+
+		updateCceAccessConfigOpt.JSONBody = utils.RemoveNil(buildUpdateCceAccessConfigBodyParams(d))
+		_, err = ltsClient.Request("PUT", updateCceAccessConfigPath, &updateCceAccessConfigOpt)
+		if err != nil {
+			return diag.Errorf("error updating cce access config: %s", err)
+		}
+	}
+	return resourceCceAccessConfigRead(ctx, d, meta)
+}
+
+func buildUpdateCceAccessConfigBodyParams(d *schema.ResourceData) map[string]interface{} {
+	bodyParams := map[string]interface{}{
+		"access_config_id":     d.Id(),
+		"access_config_detail": buildCceAccessConfigDeatilRequestBody(d.Get("access_config")),
+		"host_group_info":      buildHostGroupInfoRequestBody(d),
+		"access_config_tag":    utils.ExpandResourceTags(d.Get("tags").(map[string]interface{})),
+		"cluster_id":           d.Get("cluster_id"),
+		"binary_collect":       utils.ValueIngoreEmpty(d.Get("binary_collect")),
+		"log_split":            utils.ValueIngoreEmpty(d.Get("log_split")),
+	}
+	return bodyParams
+}
+
+func cceAccessConfigResourceImportState(_ context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	name := d.Id()
+	d.Set("name", name)
+
+	var (
+		cfg                        = meta.(*config.Config)
+		region                     = cfg.GetRegion(d)
+		listCceAccessConfigProduct = "lts"
+	)
```

Comment:
why not keep them (variables) together？
```suggestion
	var (
		cfg                        = meta.(*config.Config)
		region                     = cfg.GetRegion(d)
		listCceAccessConfigProduct = "lts"
		name = d.Id()
	)
	
	d.Set("name", name)
```

Reply:
done